### ISSUE TITLE
Apple login is not working on Android because of MFA

### ISSUE THREAD

**Issue by Tom3652** [aboutyou/dart_packages#359]
Hello and thanks for the package ! 

I am using it for my android app and i can open the Apple login web page successfully. 

However, when i enter my Apple account, there is the Apple error "An error happened..." and i receive the MFA code on my other devices to login but the Apple web page is not refreshing and showing it, so it is still on this error and we can't move on in the login process.

What can i do about it ? 

Thanks in advance

**Comment by tp** [comment]
Hey, I don't recall any such case. Is this problem persisting for you or was it maybe a temporary issue?

If it persists, would be great if you can upload a screen recording so I can fully understand the issue.

As I understand it, the initial redirect to the Apple login page works, so I think from this package's side everything is in order (if something is misconfigured for example, that should err immediately and not show the login). So maybe this was a temporary sign in issue or some issue with the test account maybe?

**Comment by Tom3652** [comment]
Hello, thanks for your answer yes sure let me try again and upload a screen recording if it's still not working.
I will also try using different accounts to see if the behavior is the same !

**Comment by Tom3652** [comment]
Oh my bad, it was indeed a temporary issue and probably on the Apple side because now i could enter the MFA code :) 
I have another error with Firebase but it's not this plugin's fault, i will dig to fix it. 

Thanks !
